src/esn.calendar.libs/app/components/event/form/event-form.controller.spec.js

```diff
@@ -2003,6 +2005,53 @@ describe('The CalEventFormController controller', function() {
       });
     });
 
+    describe('the updateLocationLink method', function() {
+      beforeEach(function() {
+        openStub = sinon.stub($window, 'open');
+      });
+
+      afterEach(function() {
+        openStub.restore();
+      });
+
+      it('should add // when the location link does not start with http', function() {
+
+        scope.event = CalendarShell.fromIncompleteShell({
+          title: 'title',
+          path: '/calendars/' + owner._id + '/' + calendarTest.id + '/eventID',
+          start: moment('2016-12-08 12:30'),
+          end: moment('2016-12-08 13:30'),
+          location: 'link.com',
+          etag: '123456',
+          alarm: {
+            trigger: '-PT1M',
+            attendee: 'test@open-paas.org'
+          }
+        });
+        initController();
+        scope.openLocationLink(scope.event.location);
+
+        expect(openStub).to.have.been.calledWith('//link.com', '_blank');
+      });
+      it('should not add // when the location start with http', function() {
```

Comment:
```suggestion

      it('should not add // when the location start with http', function() {
```

Reply:
I didn't update the ESLint config until the last time